examples/persistent_homology_graphs.ipynb

```diff
@@ -0,0 +1,659 @@
+{
```

Comment:
<ul><li>Latexify the vertex labels "edge ij has as weight the length of the shortest path from vertex i to vertex j"</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/giotto-ai/giotto-tda/pull/462/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Thanks!</p>

Reply:
<p>Done.</p>